style/styles_learn.css

```diff
@@ -0,0 +1,214 @@
+.learn {
+  background-color: #000;
+  display: flex;
+  height: 494px;
+}
+
+.learn__block-title,
+.learn__block-paragraph,
+.learn__block-link {
+  color: #fff;
+}
+
+.learn__block {
+  margin: auto 0;
+  max-width: 311px;
+}
+
+.learn__block-text {
+  width: 85%;
+  padding-top: 5px;
+}
+
+.learn__block-title {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 22px;
+  line-height: 28px;
+  margin: 0 0 12px;
+}
+
+.learn__block-paragraph {
+  font-family: "Gotham Pro Regular", sans-serif;
+  font-size: 14px;
+  line-height: 17px;
+  margin: 0 0 18px;
+}
+
+.learn__block-link {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 12px;
+  text-decoration: none;
+}
+
+.learn__block-link::after {
+  content: url("../pic/Arrow.png");
+  padding-left: 8px;
+}
+
+.learn__video-block {
+  max-width: 719px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.learn__video {
+  width: 89.57%;
+  margin-right: 6.5px;
+}
+
+@media (max-width: 1024px) {
+  .learn {
+    height: 442px;
+  }
+
+  .learn__video-block {
+    max-width: 710px;
+  }
+
+  .learn__video {
+    width: 89.3%;
+    margin-right: 8px;
+  }
+
+  .learn__block-text {
+    width: 87%;
+    padding-top: 0;
+  }
+
+  .learn__block-title {
+    margin: 0 0 7px;
+    line-height: 27px;
+  }
+
+  .learn__block-paragraph {
+    font-size: 13px;
+    margin: 0 0 10px;
+    line-height: 16.4px;
+  }
+
+  .learn__block-link::after {
+    padding-left: 13px;
+  }
+}
+
+ @media (max-width: 768px) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)